docs/examples/use_cases/tensorflow/resnet-n/nvutils/common.py

```diff
@@ -34,7 +38,10 @@ def __init__(self, rescaled_lr, step_boundaries, lr_values, warmup_steps,
     super(PiecewiseConstantDecayWithWarmup, self).__init__()
 
     self.rescaled_lr = rescaled_lr
-    self.step_boundaries = step_boundaries
+    if StrictVersion(tf.__version__) >= StrictVersion("2.13"):
+      self.step_boundaries = [int(b) for b in step_boundaries]
+    else:
+      self.step_boundaries = step_boundaries
```

Comment:
Can't we just stick to ints? Do the older versions require the number to be float?

Reply:
Unfortunately it requires. See the previous build - relevant job failed with:
```
 ValueError: Boundaries (<dtype: 'int32'>) must have the same dtype as x (<dtype: 'float32'>).```

Reply:
Thanks!